commit 2f08c0ec36332a2d0edab7b3613fccd62c891116
Author: Stu Hood <stuhood@gmail.com>
Date:   Wed Oct 28 14:20:44 2020 -0700

    Passing test.
    
    [ci skip-rust]
    
    [ci skip-build-wheels]

diff --git a/src/python/pants/backend/python/lint/isort/rules_integration_test.py b/src/python/pants/backend/python/lint/isort/rules_integration_test.py
index 80da441a8..7754e93cf 100644
--- a/src/python/pants/backend/python/lint/isort/rules_integration_test.py
+++ b/src/python/pants/backend/python/lint/isort/rules_integration_test.py
@@ -16,6 +16,7 @@ from pants.core.util_rules.source_files import SourceFiles, SourceFilesRequest
 from pants.engine.addresses import Address
 from pants.engine.fs import CreateDigest, Digest, FileContent
 from pants.engine.target import Target
+from pants.option.config import TomlSerializer
 from pants.testutil.rule_runner import QueryRule, RuleRunner
 
 
@@ -65,7 +66,10 @@ def run_isort(
         args.append("--isort-config=.isort.cfg")
     if passthrough_args:
         argsfile = "argsfile.toml"
-        rule_runner.create_file(relpath=argsfile, contents="TODO")
+        rule_runner.create_file(
+            relpath=argsfile,
+            contents=TomlSerializer({"atfile": {"args": passthrough_args}}).serialize(),
+        )
         argsfile_path = os.path.join(rule_runner.build_root, argsfile)
         args.append(f"--isort-args=@{argsfile_path}")
     if skip:
