--- node-v6.8.0.org/node.gyp  2016-10-13 01:33:29.043361487 +0000
+++ node-v6.8.0/node.gyp  2016-10-13 02:04:47.804292217 +0000
@@ -594,7 +594,7 @@
               ],
               'outputs': ['<(SHARED_INTERMEDIATE_DIR)/openssl.def'],
               'action': [
-                'python',
+                'python2.7',
                 'tools/mkssldef.py',
                 '<@(mkssldef_flags)',
                 '-o',
@@ -669,7 +669,7 @@
                 '<(SHARED_INTERMEDIATE_DIR)/v8_inspector_protocol_json.h',
               ],
               'action': [
-                'python',
+                'python2.7',
                 'tools/compress_json.py',
                 '<@(_inputs)',
                 '<@(_outputs)',
@@ -705,7 +705,7 @@
             }]
           ],
           'action': [
-            'python',
+            'python2.7',
             'tools/js2c.py',
             '<@(_outputs)',
             '<@(_inputs)',
